### PR TITLE
Added support for the Lua and Luau languages.

### PR DESCRIPTION
**Pull request by howmanysmall**
### Description

I added support for the [Lua](https://www.lua.org/) and [Luau](https://luau-lang.org/) languages.

### CODE REVIEW

File: frontend/static/languages/code_lua.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    "nil",
+    "not",
+    "or",
+    "repeat",
+    "return",
+    "then",
+    "true",
+    "until",
+    "while",
+
```
Comment (Bruception) [inline frontend/static/languages/code_lua.json:27]: Why these new lines?

File: frontend/static/languages/code_luau.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    "local",
+    "nil",
+    "not",
+    "or",
+    "repeat",
+    "return",
+    "then",
+    "true",
+    "until",
+    "while",
```
Comment (Bruception) [inline frontend/static/languages/code_luau.json:26]: Same as above

File: frontend/static/languages/code_lua.json (hunk 1 above)
Comment (howmanysmall) [reply frontend/static/languages/code_lua.json:27]: Oh, that's just me separating them based on what they'd be categorized as. First group is keywords you cannot use as a variable name, second one stuff that you can.

Comment (Bruception) [reply frontend/static/languages/code_lua.json:27]: > Oh, that's just me separating them based on what they'd be categorized as. First group is keywords you cannot use as a variable name, second one stuff that you can.

Ah, I think the prettier job will fail if you include those. It’s okay to remove them.

**Comment by Miodec** [comment]
Ill just merge this, prettier will remove the new lines.

**Pull request merged**